ymcatwincities/openy
# [OYPD-199] Fix images migration

Task: OYPD-199

### How to review

1. Go to `admin/content/media` and make sure that all media images are imported
2. Go to `/accelerator` and make sure it's working
3. Go to `/locations/downtown-ymca` and make sure it's working
